bench/SystemOutBench.java

```diff
@@ -0,0 +1,29 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+
+public class SystemOutBench {
+  void runBenchmark() {
+    long start, time;
+
+    start = JVM.monotonicTimeMillis();
+    for (int i = 0; i < 1000; i++) {
+      System.out.println("I am the very model of a modern major general.");
+    }
+
```

Comment:
You could print a result for println here and a different one for print later, so the output would be:
println: X ms
print: Y ms